README.md

```diff
@@ -48,7 +48,7 @@ print(results)
 ```
 
 ```bash
-[LoadFlowComponentResult(component_num=0, status=CONVERGED, iteration_count=3, slack_bus_id='VL4_0', slack_bus_active_power_mismatch=-0.006081)]
+[ComponentResult(connected_component_num=0, synchronous_component_num=0, status=CONVERGED, status_text=CONVERGED, iteration_count=3, reference_bus_id='VL1_0', slack_bus_results=[SlackBusResult(id='VL1_0', active_power_mismatch=-0.006730108618313579)], distributed_active_power=0.0)]
```

Comment:
the slack bus id is replaced by slack bus result ? why is it not the same ? (VL4_0/VL1_0)

Reply:
I think that the readme was previously incorrect / outdated, i.e. it is not my change. (can see also outdated because of `LoadFlowComponentResult` vs now `ComponentResult`)

Today on main branch if you follow the readme you get VL1_0, not VL4_0. 

Also:
- tests/test_loadflow.py gets VL1_0 in test_run (main branch) / renamed test_run_ac in this PR
- docs/user_guide/loadflow.rst the slack is unchanged with this PR in doctest.

Reply:
ok it works for me